cedar-wasm/src/validator.rs

```diff
@@ -14,41 +14,61 @@
  * limitations under the License.
  */
 
-use cedar_policy::ffi::{
-    validate, ValidationAnswer, ValidationCall, ValidationError, ValidationWarning,
-};
+use cedar_policy::ffi;
 use serde::{Deserialize, Serialize};
 use tsify::Tsify;
 use wasm_bindgen::prelude::*;
 
+/// `ffi::ValidationAnswer`, but with `serde(rename_all = "camelCase")`
+///
+/// REVIEW: do we want to just camelCase the fields in the normal
+/// `ValidationAnswer`, and adapt Java and others? It's confusing for
+/// Rust<->Wasm to use camelCased JSON field names but Rust<->Java to use
+/// snake_cased JSON field names
```

Comment:
No strong feelings. Would it be weird if we set up serde `alias`es to support both snake and camel case (and strange combinations thereof)?

Reply:
Various opinions on stack overflow: https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase

Reply:
Serde `alias` is a good idea, but only helps for deserialization.  Serialization still has to pick one.

Reply:
aliasing does feel a little weird IMO. I prefer picking one convention for everything

Reply:
TS folks seem to prefer camelCase in #805

Reply:
Yeah camelCase is definitely preferred in ts/js

Reply:
#814 making the global change to camelCase in JSON throughout the `ffi` module